dali/pipeline/operators/reader/loader/indexed_file_loader.h

```diff
@@ -49,13 +49,27 @@ class IndexedFileLoader : public Loader<CPUBackend, Tensor<CPUBackend>> {
       current_file_ = FileStream::Open(uris_[file_index]);
       current_file_index_ = file_index;
     }
-    tensor->Resize({size});
-    tensor->mutable_data<uint8_t>();
 
-    int64 n_read = current_file_->Read(reinterpret_cast<uint8_t*>(tensor->raw_mutable_data()),
-                        size);
+    if (!copy_read_data_) {
+      uint8_t* p = nullptr;
+      p = current_file_->Get(size);
+      DALI_ENFORCE(p != nullptr, "Error reading from a file " + uris_[current_file_index_]);
+      // Wrap the raw data in the Tensor object.
+      tensor->ShareData(p, size, {size});
+
+      TypeInfo type;
+      type.SetType<uint8_t>();
+      tensor->set_type(type);
+    } else {
+      tensor->Resize({size});
+      tensor->mutable_data<uint8_t>();
+
+      int64 n_read = current_file_->Read(reinterpret_cast<uint8_t*>(tensor->raw_mutable_data()),
+                          size);
```

Comment:
the indentation seems a bit random in this line. Are we following any rules regarding that?

Reply:
Will do

Reply:
Done (I hope)